deployments/cardano-prod.nix

```diff
@@ -6,13 +6,24 @@ let
       ./../modules/datadog.nix
       ./../modules/papertrail.nix
     ];
-    services.dd-agent.tags = ["env:production"];
 
     # DEVOPS-64: disable log bursting
     services.journald.rateLimitBurst = 0;
 
     # Initial block is big enough to hold 3 months of transactions
     deployment.ec2.ebsInitialRootDiskSize = mkForce 700;
+
+    services.dd-agent.tags = ["env:production"];
+    services.dd-agent.processConfig = ''
+      init_config:
+
+      instances:
+      - name: cardano-node
+        search_string: ['cardano-node']
+        exact_match: False
+        thresholds:
+          critical: [1, 2]
+    '';
```

Comment:
> Unfortunately, this isn't as easy as moving things around, as there are many ways to pin nixpkgs (in shell, or in Nix, or in config.yaml, etc).

We ought to standardize on one strategy.

> I have linked the nixpkgs pull request that this change was tested with as datadog changes aren't merged upstream yet.

When we're waiting on upstream changes this makes a lot of sense. However, we have to be careful to avoid stomping on each other. In the absence of a single location (per iohk-nixops branch) for setting the NIX_PATH, when there are multiple pending upstream changes we risk accidentally introducing regression.

Would it help to make an official IOHK fork of nixpkgs and coordinate changes there? Ultimately we benefit from maintaining parity with nixos/nixpkgs, so I'm not suggesting we diverge any more than necessary.